kpatch-build/kpatch-intermediate.h

```diff
@@ -26,7 +26,7 @@
 
 struct kpatch_symbol {
 	unsigned long src;
-	unsigned long pos;
+	unsigned long sympos;
```

Comment:
I don't have a very strong opinion on this, but adding "sym" to an element of a symbol feels a bit redundant.
I think the change is good for lookup_result however.

Reply:
Good point.  I also don't have a strong feeling either way but I might prefer to leave it as 'sympos' despite the redundancy, because that's what we call the value upstream.

Reply:
IMHO sympos is clearer (coming from someone familiar with the upstream code)